assets/src/patterns/605.svg

```diff
@@ -0,0 +1,146 @@
+<!-- Generator: Adobe Illustrator 22.1.0, SVG Export Plug-In  -->
+<svg version="1.1"
+     xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:a="http://ns.adobe.com/AdobeSVGViewerExtensions/3.0/"
+     x="0px" y="0px" width="53.4px" height="53.2px" viewBox="0 0 53.4 53.2" style="enable-background:new 0 0 53.4 53.2;"
+     xml:space="preserve">
+<style type="text/css">
+    .st0{fill:none;stroke:#000000;stroke-width:0.7;stroke-linecap:round;}
+    .st1{fill:none;stroke:#000000;stroke-width:0.3;}
+</style>
+<defs>
+</defs>
+<line class="st0" x1="28.4" y1="9.6" x2="28.4" y2="9.6"/>
```

Comment:
These Illustrator paths are problematic with the minifier - it's a zero-length line, that shows up as a single dot. Shape to path conversion (the bulk of SVG minification) ignores them. We'll need to modify in Illustrator to fix.